compiler/rustc_symbol_mangling/src/v0.rs

```diff
@@ -262,15 +262,16 @@ impl<'tcx> SymbolMangler<'tcx> {
     fn print_pat(&mut self, pat: ty::Pattern<'tcx>) -> Result<(), std::fmt::Error> {
         Ok(match *pat {
             ty::PatternKind::Range { start, end } => {
-                let consts = [start, end];
-                for ct in consts {
-                    Ty::new_array_with_const_len(self.tcx, self.tcx.types.unit, ct).print(self)?;
-                }
+                self.push("R");
+                self.print_const(start)?;
+                self.print_const(end)?;
             }
             ty::PatternKind::Or(patterns) => {
+                self.push("O");
```

Comment:
`O` is already [used by](https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/symbol-mangling/v0.md#mut-ptr-type) mutable raw pointers. Should we pick a different character?

Reply:
this is nested below `W`, never at the top level. I think it is unambiguous